### PR TITLE
[IMP] product_last_price_info: French translation.

### PR DESCRIPTION
**Pull request by clonedagain**

### CODE REVIEW

File: product_last_price_info/i18n/fr.po [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+
+#. module: product_last_price_info
+#: field:product.product,last_sale_price:0
+msgid "Last sale price"
+msgstr "Prix dernière vente"
+
+#. module: product_last_price_info
+#: view:product.product:product_last_price_info.product_form_view_inh_lastpurchasesaleinfo
+msgid "Procurements"
+msgstr ""
```
Comment (oihane) [inline product_last_price_info/i18n/fr.po]: I think it will be `Approvisionnements`

**Comment by clonedagain** [comment]
Thanks @oihane I had missed this line.

**Comment by pedrobaeza** [comment]
For me it's OK, but I have no idea of french, sorry.

@oihane, can we merge?

**Comment by oihane** [comment]
:+1:

**Pull request merged**